aporia.nim

```diff
@@ -1686,23 +1686,58 @@ proc extraBtn_Clicked(button: PButton, user_data: Pgpointer) =
 proc goLine_Changed(ed: PEditable, d: Pgpointer) =
   var line = win.goLineBar.entry.getText()
   var lineNum: BiggestInt = -1
-  if parseBiggestInt($line, lineNum) != 0:
+  var column: BiggestInt = 0
+  var validLine = 0
+  var validColumn = 0
+  var columnEntered = false
+
+  if ($line).count(":") == 1:
+    var lineComponents = ($line).split(":")
+    validLine = parseBiggestInt(lineComponents[0].strip(), lineNum)
+    validColumn = parseBiggestInt(lineComponents[1].strip(), column)
+    columnEntered = true
+  elif ($line).count(":") == 0:
+    validLine = parseBiggestInt(($line).strip(), lineNum)
```

Comment:
Instead of using split do something like this:

```nim
i = 0
i += parseBiggestInt(line, lineNum, i)
if line[i] == ':':
  i.inc # skip ':'
  i += parseBiggestInt(line, columnNum, i)
```


```